### DISCUSSION TITLE
[Witches' Eve] Feathers x Shop

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2743] [General]
We are moving to a real game now and do you guys do that? Are we going to play or shop?

Are you sure you're going to put those two things together?

You are practically forcing players to shop in the store to get better in-game items, what feeling do you think this brings to current players and new players?

Hope you rethink about it.

Thank you.

**Comment by TheDcol** [comment]
Hey, the decision to hand out feathers with merch has been reverted :)
